docker-storage-setup.sh

```diff
@@ -720,6 +719,21 @@ unit: sectors
 
 ${dev}1 : start=     2048, size=  ${size}, Id=8e
 EOF
+}
+
+create_partition_parted(){
+  local dev="$1"
+  parted $dev --script mklabel msdos mkpart primary 0% 100%
```

Comment:
I forgot the keyword `on` at the end. So it should be.

arted $dev --script mklabel msdos mkpart primary 0% 100% set 1 lvm on